docs/theming.md

```diff
@@ -185,9 +185,174 @@ extension MyViewController {
 ```
 <!--</div>-->
 
+## Migration guide: Themers to Theming extensions
+
+This migration guide covers the typical migration path from Themer usage to Theming extensions.
+
+Theming extensions are discussed in detail above. Themers are discussed in the [Themers](#themers)
+section. Themers will eventually be deprecated and deleted. Theming extensions are the recommended
+replacement.
```

Comment:
Please move these last two sentences to the first paragraph. They are good introductory information about the what and why of this section.